src/mame/juku/juku.cpp

```diff
@@ -1,29 +1,30 @@
 // license: BSD-3-Clause
-// copyright-holders: Dirk Best
+// copyright-holders: Dirk Best, Märt Põder
 /***************************************************************************
 
-    Juku E5101
+    Juku E5101/E5104
 
     Hardware:
-    - КР580ВМ80A
+    - КР580ВМ80A (called КР580ИК80A until 1986)
     - КР580ИР82
-    - КР580ВА86
-    - КР580ВА87
+    - КР580ВА86 x3
+    - КР580ВА87 x3
     - КР580ВИ53 x3
     - КР580ВК38
     - КР580ВН59
     - КР580ВВ51A x2
     - КР580ВВ55A x2
+    - КР1818ВГ93 (on all E5104 production models)
```

Comment:
Would it make sense to create two drivers, `e5101` and `e5104`, with only the latter including the floppy drive emulation (I assume the 5101 uses a tape)?